src/advanced/bitcoin-anchoring.md

```diff
@@ -70,65 +69,42 @@ floor(2/3*N) + 1` is used as a supermajority.
 
-### Transaction Malleability
+### Transaction Malleability and SegWit
 
 Validators signatures are openly written in the Exonum blockchain; more
-than `M` signatures can be published (it is a common case). There
-is a special algorithm selecting `M` signatures deterministically. If
-all validators are legitimate, certain transactions are built
-unequivocally.
-
-However, any Byzantine node can step out from the deterministic algorithm and
-use another signature list for the anchoring transaction. This node may create
-a transaction with the same anchored hash (the same data-output) but another
-tx-id, and broadcast it to the Bitcoin network. Such non-standard
-transactions present a problem because a new anchoring transaction may be built
-even if the previous one has not been included yet in any Bitcoin block.
-However, there is a chance that this previous transaction or one of its
-ancestors was mutated by a malicious validator as described above, and the
-mutated transaction has been committed to the Bitcoin blockchain. This
-would make the original transaction ineligible for inclusion into the
-Bitcoin blockchain.
-
-To handle this problem, the consensus is used to select the appropriate previous
-transaction via [LECTs](#lect).
-
-### LECT
-
-Every validator defines which anchoring transaction is considered to be
-the latest one. This transaction should be spent in the new anchoring
-transaction and is called the Latest Expected
-Correct Transaction (LECT). LECTs of all validators are published in the
-Exonum blockchain. While creating a new anchoring transaction, the
-supermajority of validators select a common LECT and spend its change output.
-
-Every validator refreshes its LECT with a [custom
-schedule](#lect-updating-interval). To get a new LECT, the validator uses the
-API of a [Bitcoin node](#bitcoind-node). The new LECT must have the following
-properties:
-
-- It is a valid anchoring transaction for the current Exonum blockchain;
-- It may have any amount of confirmations, in particular, 0;
-- Its change output should be unspent. That means that the specified
-  validator believes there was no following anchoring transactions after
-  this one;
-- Among all Bitcoin transactions satisfying the previous properties,
-  LECT should have the greatest anchored Exonum block height;
-- If multiple transactions meet the previous requirements, any of them
-  may be chosen as the LECT.
-
-The LECT solves the transaction malleability problem, though anchoring
-transactions sometimes may be orphaned and never written to the Bitcoin
-blockchain. However, it is safe enough as the following anchoring
-transactions effectively anchor all previous ones.
-
-Exonum uses a [`bitcoind` client](#bitcoind-node), and only one of the
-transactions that meet the requirements can be considered valid by the
-bitcoind-node.
+than `M` signatures can be published (it is a common case). Previously, as these
```

Comment:
TBH, I don't think that such a detailed introduction into the previous state of affairs is necessary here. IMO, this paragraph could be safely removed.